File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoCase/src/pato/Pato_Bravo.java
package pato;

public class Pato_Bravo extends Pato implements  Padrao_Grasnar{

	public Pato_Bravo()	
	{
		setComportamento(new Voar_Foguete());		
	}
	
	public String mostrar() {		
		return "Eu sou o Pato Bravo.";
	}

	public String grasnar() {
		return "Que-Que. Grrrrrrrrr.";
	}

}

File: ADS-2-Semestre/EngenhariaDeSoftware/InterfacePatoC#/Pato/PatoBravo.cs
using Pato;
using System;
using System.Collections.Generic;
using System.Text;

namespace JogoPato
{
	public class PatoBravo : Pato, IPadraoGrasnar
    {
		public PatoBravo()
		{
			setComportamento(new VoaveisAsa());
		}

		public String grasnar()
		{
			return "Que-Que. Grrrrrrrrr.";
		}

		public override string mostrar()
		{
			return "Eu sou o Pato Bravo.";
		}
	}
}
